README.md

```diff
@@ -29,28 +29,86 @@ Check out the [getting started guide](./docs/getting_started.md) for launching a
 
-**NOTES:**
-* We aim to cut a CAPV minor release approximately one week after the corresponding CAPI minor release is out.
-* We aim to cut a CAPV minor or patch release with support for a new Kubernetes minor version approximately 3 business days after releases for CAPI and CPI that support the new Kubernetes version are available.
+- The first eight months of this timeframe will be considered the standard support period for a minor release.
+- The next four months the minor release will be considered in maintenance mode.
+- At the end of the four-month maintenance mode period, the minor release will be considered EOL (end of life) and cherry picks to the associated branch are to be closed soon afterwards.
+
+The table below documents support matrix for Cluster API provider vSphere versions (older versions omitted).
+
+| Minor Release | Status                  | Supported Until (including maintenance mode)                                         |
+|---------------|-------------------------|--------------------------------------------------------------------------------------|
+| v1.14.x       | Standard support period | in maintenance mode when v1.16.0 will be released, EOL when v1.17.0 will be released |
+| v1.13.x       | Standard support period | in maintenance mode when v1.15.0 will be released, EOL when v1.16.0 will be released |
+| v1.12.x       | Maintenance mode        | Maintenance mode v1.14.0 release date, EOL when v1.15.0 will be released             |
+| v1.11.x       | EOL                     | EOL since v1.14.0 release date                                                       |
+| v1.10.x       | EOL                     | EOL since v1.13.0 release date                                                       |
+| v1.9.x        | EOL                     | EOL since v1.12.0 release date                                                       |
+
+## Compatibility with Cluster API
+
+Only a limited set of combinations between releases of CAPV and Cluster API are tested on CI and thus actively
+supported:
+
+| CAPV version | CAPI version used in CI |
+|--------------|:-----------------------:|
+| CAPV v1.12   |        CAPI v1.9        |
+| CAPV v1.13   |       CAPI v1.10        |
+| CAPV v1.14   |       CAPI v1.11        |
+
+Other CAPV/CAPI version combinations might work, but it is up to the users to validate those combinations; 
+also please be aware that in case of a CAPI/CAPV combination not listed above, support will be provided at best effort.
+
+Following info should be used when identifying possible CAPV/CAPI version combinations outside the table above:
+
+Compatibility between CAPV and Cluster API depends on which Cluster API contract versions is supported by CAPI and by
+which contract version is implemented in CAPV.
+
+The following table documents the Cluster API contract versions implemented by supported CAPV versions:
+
+|            | Cluster API contract version v1beta1 | Cluster API contract version v1beta1 |
```

Comment:
```suggestion
|            | Cluster API contract version v1beta1 | Cluster API contract version v1beta2 |
```

Reply:
Entirely up to you but maybe just merge the two columns and write v1beta1 / v1beta2 in the cell like in l.98++

But fine as is for me